<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_station</class>
 <widget class="QWidget" name="Add_station">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #000000; /* الخلفية الرئيسية سوداء */
}
QPushButton:pressed, QPushButton:checked {
    background-color: #44bd32; /* green when clicked */
    color: white;}
/* Sidebar */
QListWidget {
    background-color: #1e1e1e;
    color: #ffffff;
    border-radius: 8px;
    font-size: 12pt;
}
QListWidget::item:selected {
    background: #00a8ff;
    color: white;
}
QListWidget::item:hover {
    background: #333333;
}

/* Tabs */
QTabBar::tab {
    font-size: 12pt;
    padding: 8px 20px;
    margin: 4px;
    border-radius: 12px;
    background: #1e1e1e;
    color: #bbbbbb;
}
QTabBar::tab:selected {
    background: #44bd32;
    color: white;
    font-weight: bold;
}
QTabBar::tab:hover {
    background: #4cd137;
    color: white;
}

/* Forms (LineEdits, Labels) */
QLabel {
    color: #ffffff;
    font-size: 11pt;
}
QLineEdit {
    font-size: 11pt;
    padding: 6px;
    border: 1px solid #444444;
    border-radius: 6px;
    background: #1e1e1e;
    color: #ffffff;
}
QLineEdit:focus {
    border: 2px solid #44bd32;
    background: #2c2c2c;
    color: #ffffff;
}

/* Buttons */
QPushButton {
    font-size: 11pt;
    font-weight: bold;
    padding: 8px 16px;
    border-radius: 8px;
    color: white;
    background-color: #1e1e1e;
    border: 1px solid #444;
}
QPushButton#add {
    background-color: #44bd32;
}
QPushButton#add:hover {
    background-color: #4cd137;
}
QPushButton#update {
    background-color: #0097e6;
}
QPushButton#update:hover {
    background-color: #00a8ff;
}
QPushButton#delete {
    background-color: #e84118;
}
QPushButton#delete:hover {
    background-color: #c23616;
}
/* ---------------- Buttons ---------------- */
QPushButton:pressed, QPushButton:checked {
    background-color: #44bd32; /* green when clicked */
    color: white;
}

/* ---------------- LineEdits ---------------- */
QLineEdit:focus {
    border: 2px solid #44bd32; /* green border when focused */
    background: #2c2c2c;
    color: #ffffff;
}

/* ---------------- QListWidget Items ---------------- */
QListWidget::item:selected {
    background-color: #44bd32; /* green when selected */
    color: white;
}

/* ---------------- QTabBar Tabs ---------------- */
QTabBar::tab:selected {
    background-color: #44bd32; /* green for selected tab */
    color: white;
    font-weight: bold;
}
</string>
  </property>
  <widget class="QLineEdit" name="stationNameInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Station&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Student ID</string>
   </property>
   <property name="Sname" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="latitudeInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Student ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="longitudeInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Student ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addStationButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Station</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
